noxfile.py

```diff
@@ -159,13 +124,9 @@ def mypy(session):
     # Ensure that mypy itself ran successfully
     assert process.returncode in (0, 1)
 
-    for line in process.stdout.split("\n"):
```

Comment:
Can we change all this logic into a `session.run(mypy --strict src/)`?

Reply:
What kind of change do you mean?

Reply:
Instead of capturing the output of mypy and doing all this filtering we now don't expect any mypy errors so in theory can use `session.run()` directly?